tasks/main.yml

```diff
@@ -58,16 +60,19 @@
     src: manage_certs.conf
     dest: "{{ LOAD_BALANCER_MANAGE_CERTS_CONF }}"
 
-- name: copy config watcher systemd service file
+- name: copy config and cert watcher systemd service files
   template:
-    src: haproxy-config-watcher.service
-    dest: /lib/systemd/system/haproxy-config-watcher.service
-
-- name: enable and start config watcher
-  command: systemctl {{ item }} haproxy-config-watcher
+    src: "{{ item }}"
+    dest: "/lib/systemd/system/{{ item }}"
```

Comment:
After deployment this should work fine (it does work for the integration test server).